models/issue.go

```diff
@@ -1440,6 +1459,9 @@ func GetUserIssueStats(opts UserIssueStatsOptions) (*IssueStats, error) {
 	if len(opts.RepoIDs) > 0 {
 		cond = cond.And(builder.In("issue.repo_id", opts.RepoIDs))
 	}
+	if len(opts.IssueIDs) > 0 {
+		cond = cond.And(builder.In("issue.id", opts.IssueIDs))
```

Comment:
Be careful! There's a limit to the number of arguments an SQL statement accepts, and it depends on the engine.

Reply:
Understood. The same condition is done a couple other times in issue.go (in existing code) like in the GetIssueStats method. Is there somewhere in gitea that breaks up the queries into batches I could look at as an example? But I'd assume if it is a problem here, it would be a problem in the other places.

Reply:
@guillep2k I added some code to limit the in clause to 1000 ids. I didn't see a limit for MySql or Postgres, but 1000 seems to be a reasonable limit as far as I can tell.